<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAliasListPage</class>
 <widget class="QWidget" name="MyAliasListPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Aliases</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Refresh alias list</string>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newAlias">
       <property name="toolTip">
        <string>Create a new alias</string>
       </property>
       <property name="text">
        <string>New Alias</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyAlias">
       <property name="toolTip">
        <string>Copy the currently selected alias to the system clipboard</string>
       </property>
       <property name="text">
        <string>Copy Alias ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="toolTip">
        <string>Edit selected alias</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="transferButton">
       <property name="toolTip">
        <string>Transfer selected alias</string>
       </property>
       <property name="text">
        <string>Transfer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newPubKey">
       <property name="toolTip">
        <string>Create a new public key used for transferring aliases</string>
       </property>
       <property name="text">
        <string>New Public Key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="whitelistButton">
       <property name="toolTip">
        <string>Open list of affiliates associated with this alias</string>
       </property>
       <property name="text">
        <string>Affiliations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signMultisigButton">
       <property name="text">
        <string>Sign Multisig Tx</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="toolTip">
        <string>Export the data in the current tab to a file</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../syscoin.qrc"/>
 </resources>
 <connections/>
</ui>
